.github/workflows/system_tests.yaml

```diff
@@ -14,17 +14,15 @@ jobs:
           path: C:\
       - uses: actions/checkout@v4
       - run: mkdir C:\managed_robots
-      - run: cd data\retry_rcc\
-      - run: tar -c -a -f C:\managed_robots\retry_rcc_defN.zip .
-      - run: cd ..\..
+      - run: tar -c -a -C tests\minimal_suite\ -f C:\managed_robots\minimal_suite.zip *
```

Comment:
Rule of Sven: In scripts, always use the long form of the arguments, i.e.,
-a --> --auto-compress
-c --> --create
-C --> --directory
-f --> --file